templates/default/knative-serving.yml.erb

```diff
@@ -601,10 +601,10 @@ data:
     kubernetes.podspec-affinity: "disabled"
 
     # Indicates whether Kubernetes nodeSelector support is enabled
-    kubernetes.podspec-nodeselector: "disabled"
+    kubernetes.podspec-nodeselector: "enabled"
 
     # Indicates whether Kubernetes tolerations support is enabled
-    kubernetes.podspec-tolerations: "disabled"
+    kubernetes.podspec-tolerations: "enabled"
```

Comment:
This is the example configuration. Please add these lines immediately after `data:` (outside `_example:`) https://github.com/logicalclocks/kube-hops-chef/blob/24cb9e9cd56d8698d74e42f862ec242f597d2e00/templates/default/knative-serving.yml.erb#L580